### PR TITLE
Upgrade to v1 of the genomics API.

### PR DESCRIPTION
**Pull request by deflaux**

### CODE REVIEW

File: src/main/java/com/google/cloud/genomics/utils/Paginator.java [hunk 1]
```diff
@@ -203,9 +204,9 @@ private Datasets(Genomics genomics) {
       super(genomics);
     }
 
-    @Override Genomics.Datasets.List createSearch(Genomics.Datasets api, Long request,
+    @Override Genomics.Datasets.List createSearch(Genomics.Datasets api, String request,
```
Comment (melissachang) [inline src/main/java/com/google/cloud/genomics/utils/Paginator.java]: request -> projectId

Comment (deflaux) [inline src/main/java/com/google/cloud/genomics/utils/Paginator.java]: @melissachang I renamed it.  createSearch exists many times in that file though.  Can you PTAL and let me know if I understood you correctly?

Comment (melissachang) [inline src/main/java/com/google/cloud/genomics/utils/Paginator.java]: I was referring to Paginator.java line 280. But looking closer at the code, the argument passed into operations.list isn't used for anything. How about something like this?

    @Override Genomics.Operations.List createSearch(Genomics.Operations api, 
          Optional<String> pageToken) throws IOException {		          Optional<String> pageToken) throws IOException {
       return api.search(pageToken		 +      return pageToken.isPresent() ? api.list(request).setPageToken(pageToken.get())
           .transform(		 +          : api.list("unused argument");

Comment (deflaux) [inline src/main/java/com/google/cloud/genomics/utils/Paginator.java]: The method signature needs to include a RequestT parameter per https://github.com/deflaux/utils-java/blob/master/src/main/java/com/google/cloud/genomics/utils/Paginator.java#L996

@melissachang Is that String value really unused? https://developers.google.com/resources/api-libraries/documentation/genomics/v1/java/latest/com/google/api/services/genomics/Genomics.Operations.List.html says "Lists operations that match the specified filter in the request."

**Comment by melissachang** [comment]
LGTM

**Pull request merged**